client/my-sites/themes/themes-selection.jsx

```diff
@@ -280,10 +280,25 @@ class ThemesSelectionWithPage extends React.Component {
 		page: 1,
 	};
 
+	componentDidUpdate( nextProps ) {
+		if (
+			nextProps.search !== this.props.search ||
+			nextProps.tier !== this.props.tier ||
+			! isEqual( nextProps.filter, this.props.filter ) ||
```

Comment:
I was left with the impression that `filter` and `vertical` are strings as well, so we should be able to compare them with strict equality as well?

Reply:
Good catch, it was updated here: https://github.com/Automattic/wp-calypso/pull/68418/commits/8b2d23f49c6264114bbaafd00cea184a446b0d91